templates/data/summary.html

```diff
@@ -52,72 +52,77 @@
-      {% endfor %}
-    </table>
+      <table class="table table-striped table-bordered">
+        <thead><th style="width: 30%;">Moods</th><th style="width: 25%;">value</th><th>probability</th></thead>
+        {% for row in summary.moods %}
+          <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
+        {% endfor %}
+      </table>
 
-    <table class="table table-striped table-bordered">
-      <thead><th style="width: 30%;">Genres</th><th style="width: 25%;">value</th><th>probability</th></thead>
-      {% for row in genres %}
-        <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
-      {% endfor %}
-    </table>
+      <table class="table table-striped table-bordered">
+        <thead><th style="width: 30%;">Genres</th><th style="width: 25%;">value</th><th>probability</th></thead>
+        {% for row in summary.genres %}
+          <tr><td>{{ row[0] }}</td><td>{{ row[1] }}</td><td>{{ row[2] }}</td></tr>
+        {% endfor %}
+      </table>
 
-  {% else %}
-    <div class="alert alert-warning">
-      Sorry, we have not had a chance to calculate the high-level data for
-      this track yet. It should be calculated soon. In the meantime, have a
-      look at our <a href="{{ url_for('data.data') }}#sample-data">sample data</a>.
-    </div>
-  {% endif %}
+    {% else %}
+      <div class="alert alert-warning">
+        Sorry, we have not had a chance to calculate the high-level data for
+        this track yet. It should be calculated soon. In the meantime, have a
+        look at our <a href="{{ url_for('data.data') }}#sample-data">sample data</a>.
+      </div>
+    {% endif %}
 
-  <p>Inspect the data the above is taken from:</p>
-  <ul>
-    <li>
-      <a href="{{ url_for('data.view_low_level', mbid=mbid) }}">Low-level data</a>
-      (<a href="{{ url_for('api.get_low_level', mbid=mbid) }}">raw JSON</a>)
-    </li>
-    {% if other -%}
+    <p>Inspect the data the above is taken from:</p>
+    <ul>
       <li>
-        <a href="{{ url_for('data.view_high_level', mbid=mbid) }}">High-level data</a>
-        (<a href="{{ url_for('api.get_high_level', mbid=mbid) }}">raw JSON</a>)
+        <a href="{{ url_for('data.view_low_level', mbid=mbid) }}">Low-level data</a>
+        (<a href="{{ url_for('api.get_low_level', mbid=mbid) }}">raw JSON</a>)
       </li>
-    {%- endif %}
-  </ul>
+      {% if other -%}
+        <li>
+          <a href="{{ url_for('data.view_high_level', mbid=mbid) }}">High-level data</a>
+          (<a href="{{ url_for('api.get_high_level', mbid=mbid) }}">raw JSON</a>)
+        </li>
+      {%- endif %}
+    </ul>
+    {% else %}
+    <div class="alert alert-warning">   
+          Sorry, but we don't have more information for this track. You have this track on your hard drive? Great !! You can be the first one to contribute for this track. You just need to run a very simple extractor. Find more information <a href="{{ url_for('index.contribute') }}">here</a>. 
```

Comment:
I am not sure if we should we keep the message to be very formal. A little informal touch to the message can be quite attractive to the user. But again, I am not very sure if we want something very formal or informal.